checkers/testdata/_integration/check_main_only/linttest.golden

```diff
@@ -49,4 +49,4 @@ exit status 1
 ./main.go:208:1: unlabelStmt: label loop is redundant
 ./main.go:216:11: unlambda: replace `func(x int) int { return add1(x) }` with `add1`
 ./main.go:219:39: unslice: could simplify xs[:] to xs
-./main.go:250:6: weakCond: suspicious `xs == nil || xs[0] == 0`; nil check may not be enough, check for len
\ No newline at end of file
+./main.go:250:6: weakCond: suspicious `xs == nil || xs[0] == 0`; nil check may not be enough, check for len
```

Comment:
Hmm, it looks like a trailing newline breaks our golden tests.
I'll send a fix that solves that so your code can pass the tests.